### PR TITLE
feat: autocomplete suggestion

### PR DESCRIPTION
**Pull request by 120EE0692**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes 

### Description
image

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: client/src/components/marginals/DesktopNavbar.js [hunk 1]
```diff
@@ -32,54 +49,104 @@ const DesktopNavbar = () => {
     }
   };
 
+  const searchActive = () => {
+    setIsSearchActive((current) => !current);
+    console.log(isSearchActive);
```
Comment (rutajdash) [inline client/src/components/marginals/DesktopNavbar.js:54]:
```suggestion
```

**Review by rutajdash — CHANGES_REQUESTED** [review]

**Comment by rutajdash** [comment]
@120EE0692 please resolve conflicts

**Review by rutajdash — COMMENTED** [review]
There are formatting issues flagged in the checks.

**Comment by 120EE0692** [comment]
> There are formatting issues flagged in the checks.

@rutajdash it seems to be something else
image

**Comment by rutajdash** [comment]
> > There are formatting issues flagged in the checks.
> 
> @rutajdash it seems to be something else image

Check the prettier output stage. The script fails when it finds formatting errors.

**Comment by rutajdash** [comment]
image

There may be more files

**Comment by 120EE0692** [comment]
> image
> 
> There may be more files

image
@rutajdash no formatting issue.

**Pull request closed without merging**